Sources/SwiftParser/Declarations.swift

```diff
@@ -281,9 +281,11 @@ extension Parser {
         )
       }
 
-      let isProbablyFuncDecl = self.at(.identifier, .wildcard) || self.at(anyIn: Operator.self) != nil
+      let isPossibleFuncIdentifier = self.at(.identifier, .wildcard)
+      let isPossibleFuncParen = self.peek(isAt: .identifier, .leftParen) || self.peek(isAt: .identifier, .leftAngle)
```

Comment:
You should be able to shorten this to

```suggestion
      let isPossibleFuncParen = self.peek(isAt: .identifier, .leftParen, .leftAngle)
```

Reply:
Thanks! I shortened it. However, while testing it out, I found that in
```
class MyClass { 
    foo<Int>
}
```
`<` is parsed as a binaryOperator rather than a left angle. I think this might need some follow-up item that should be looked at.